src/cli/fleet.toit

```diff
@@ -471,6 +472,14 @@ class Fleet:
     new-file := fleet-file_.with --recovery-urls=[]
     new-file.write
 
+  recovery-info -> ByteArray:
+    broker.server-config.fill-certificate-ders: certificate-roots.MAP[it].raw
+    certificate-der/ByteArray? := null
+    json-config := broker.server-config.to-json
```

Comment:
Can we end up with something that has a ders/ders64 section here? Hmm. That wouldn't work for the device side.